server/form-pages/apply/access-and-healthcare/accessNeeds.ts

```diff
@@ -0,0 +1,126 @@
+    },
+    religiousAndCulturalNeeds: {
+      question: `Does ${this.application.person.name} have any religious or cultural needs?`,
+      furtherDetails: `Details of religious or cultural needs`,
+    },
+    interpreter: {
+      question: `Does ${this.application.person.name} need an interpreter?`,
+      language: 'Which language is an interpreter needed for?',
+    },
+    careActAssessmentCompleted: 'Has a care act assessment been completed?',
+  }
+
+  body: {
+    additionalNeeds: AdditionalNeed[]
+    religiousOrCulturalNeeds: YesOrNo
+    religiousOrCulturalNeedsDetails: string
+    careActAssessmentCompleted: YesNoOrIDK
+    needsInterpreter: YesOrNo
+    interpreterLanguage: string
+  }
+
+  constructor(body: Record<string, unknown>, private readonly application: Application) {
+    this.body = {
+      additionalNeeds: body.additionalNeeds ? ([body.additionalNeeds].flat() as Array<AdditionalNeed>) : [],
+      religiousOrCulturalNeeds: body.religiousOrCulturalNeeds as YesOrNo,
+      religiousOrCulturalNeedsDetails: (body.religiousOrCulturalNeedsDetails as string) || '',
+      needsInterpreter: body.needsInterpreter as YesOrNo,
+      interpreterLanguage: body.interpreterLanguage as string,
+      careActAssessmentCompleted: body.careActAssessmentCompleted as YesNoOrIDK,
+    }
+  }
+
+  previous() {
+    return ''
+  }
+
+  next() {
+    return ''
+  }
+
+  response() {
+    const response = {
+      [this.title]: {
+        [this.questions.needs.question]: this.body.additionalNeeds
+          .map((need, i) => (i < 1 ? additionalNeeds[need] : additionalNeeds[need].toLowerCase()))
+          .join(', '),
+        [this.questions.religiousAndCulturalNeeds.question]: pascalCase(this.body.religiousOrCulturalNeeds),
+        [this.questions.religiousAndCulturalNeeds.furtherDetails]: this.body.religiousOrCulturalNeedsDetails,
+        [this.questions.interpreter.question]: pascalCase(this.body.needsInterpreter),
+        [this.questions.careActAssessmentCompleted]:
+          this.body.careActAssessmentCompleted === 'yes' || this.body.careActAssessmentCompleted === 'no'
+            ? pascalCase(this.body.careActAssessmentCompleted)
+            : "I don't know",
+      },
+    }
+    if (this.body.needsInterpreter === 'yes') {
+      return {
+        [this.title]: {
+          ...response[this.title],
+          [this.questions.interpreter.language]: this.body.interpreterLanguage,
+        },
+      }
+    }
+    if (this.body.religiousOrCulturalNeeds === 'yes') {
+      return {
+        [this.title]: {
+          ...response[this.title],
+          religiousOrCulturalNeedsDetails: this.body.religiousOrCulturalNeedsDetails,
+        },
+      }
+    }
+
+    return response
+  }
+
+  errors() {
+    const errors: TaskListErrors<this> = {}
+
+    if (!this.body.additionalNeeds.length) {
+      errors.additionalNeeds = `Please confirm whether ${this.application.person.name} has any additional needs`
```

Comment:
I think error messages we've used in the last lean more towards `You must`, rather than `Please`.

Reply:
Good point